site/pages/gc-subway-index.hbs

```diff
@@ -0,0 +1,106 @@
+---
+{
+"title": "GC-Subway Index",
+"language": "en",
+"altLangPrefix": "gc-subway-index",
+"breadcrumb": [
```

Comment:
Add this property here above "breadcrumb":

`"pageclass": "page-type-nav",`

Because by default, the templater will take the "title" attribute as an automatically rendered `<h1>`, which gives the 2 `<h1>` that I mentioned like so:

<img width="468" alt="Screen Shot 2020-12-04 at 9 49 57 AM" src="https://user-images.githubusercontent.com/4353199/101177954-6369f180-3616-11eb-98b8-17b08f7aeb09.png">